chart/README.md

```diff
@@ -541,14 +541,63 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`.
```

Comment:
need an explanation of default values

Reply:
specifically I mean that cpu is enabled by default and memory is now and they can disable either or based on preference

Reply:
This is explained in the table below.

Reply:
May be let's add something like 

```
please refer to the "Chart Configuration Options" table below for detailed parameters for adjusting HPA configuration to match your needs.
```

Reply:
Commit. bd13d896fa8897391292795cda032af651c5ebdd